lib/vty.c

```diff
@@ -3301,9 +3301,9 @@ static void vty_mgmt_session_created(uintptr_t lib_hndl, uintptr_t usr_data,
 		return;
 	}
 
-	zlog_err("%s session for client %llu successfully!",
-		 create ? "Created" : "Destroyed",
-		 (unsigned long long)client_id);
+	zlog_debug("%s session for client %llu successfully!",
+		   create ? "Created" : "Destroyed",
+		   (unsigned long long)client_id);
```

Comment:
Can we remove the exclamation points from most these messages? Certainly in the success case we're not "SHOCKED!" right?

Reply:
Sure will remove the exclamations from the success cases.